### PR TITLE
Video Page Styling/Date Uploaded Field

### LINKED ISSUE #105: Styling for Mentor Portal Videos
#### Description
Let's try to make Mentor Videos look a little more like the one in [figma]()

#### Screenshots
This is how it looks like so far:
image

This is how we want it to be:
image

#### Tasks
- [ ] Style
- [ ] Test

### PR DESCRIPTION
**Pull request by Leundai**
## Description :star2:

Fixes #105 
I've also noticed that I forgot to add the date_uploaded field to the backend/MongoDB so I will also be adding that to this PR so that it works properly. I'll change the fields of MongoDB too once we merge this PR.

image

## Important!
The backend will break if I add the fields. So that is why your frontend might not work since it is missing some fields in the database, specifically "date_uploaded"
image
As I mentioned before once we have this approved I'll add the fields. You can do the thing above to test it out though.

## TODOs :star:
- [x] Screenshots
- [x] Testing

### CODE REVIEW

**Review by AngelaLuo49021 — CHANGES_REQUESTED** [review]
Can you bold the video title and increase the space between the video icon and the text a little bit? Also if the pin could be moved a bit more away from the border that would be great but not a big deal
image
image

**Review by AngelaLuo49021 — APPROVED** [review]
Very cool style changes, looks good!

File: frontend/src/components/pages/Videos.js [hunk 1]
```diff
@@ -24,8 +24,15 @@ function Videos() {
       const mentor = await fetchMentorByID(newMentorID);
       setName(mentor.name);
```
Comment (chyku) [inline frontend/src/components/pages/Videos.js:25]: should include this in the if (mentor) block- if there is no mentor mentor.name is also nothing?

File: frontend/src/components/pages/Videos.js [hunk 2]
```diff
@@ -24,8 +24,15 @@ function Videos() {
...
       if (mentor) {
-        setVideos(mentor.videos);
-        setFiltered(mentor.videos);
+        let videos = mentor.videos;
+        // Formats all dates from objects (in Epoch) to MongoDB parseable
+        for (let i = 0; i < videos.length; i++) {
+          const date = videos[i].date_uploaded.$date;
+          videos[i].date_uploaded = moment(date).format();
+        }
```
Comment (chyku) [inline frontend/src/components/pages/Videos.js:32]: why is it necessary to use a for loop instead of foreach?

**Review by chyku — APPROVED** [review]

**Pull request merged**